nx584/config.yaml

```diff
@@ -11,3 +14,37 @@ arch:
   - i386
 ports:
   5007/tcp: 5007
+options:
+  serial:
+   enabled: true
+   device: "/dev/serial/by-id/usb-Prolific_Technology_Inc._USB-Serial_Controller-if00-port0"
+   baud: "9600"
+  socat:
+   enabled: false
+   host: "192.168.30.1"
+   port: "10000"
+  config:
+   max_zone: "4"
+   zone_name_update: false
+   euro_date_format: true
+   use_binary_protocol: false
+   idle_time_heartbeat_seconds: "120"
+   zones: "NX Main entrance,NX Kitchen,NX Despacho,NX Garage"
```

Comment:
Also here: "Area 1, Area 2, ..."?

Reply:
I plan to do that, but it is still on development...